gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
I don't believe so. A legitimate Sonatype login and a published key pair is required.